### PR TITLE
Fix some for AppVoyer fails

### PR DESCRIPTION
**Pull request by mizunashi-mana**

### CODE REVIEW

File: test/Hpack/ConfigSpec.hs [hunk 1]
```diff
@@ -327,7 +327,7 @@ spec = do
           dependencies: ghc-prim
           baz: 42
         |]
-        (`shouldBe` [
+        (`shouldMatchList` [
```
Comment (sol) [inline test/Hpack/ConfigSpec.hs]: I pushed this to `master`.

File: src/Hpack/Util.hs [hunk 2]
```diff
@@ -159,3 +160,6 @@ type Hash = String
 
 sha256 :: String -> Hash
 sha256 c = show (hash (encodeUtf8 c) :: Digest SHA256)
+
+linesForAll :: String -> [String]
+linesForAll = lines . filter (/= '\r')
```
Comment (sol) [inline src/Hpack/Util.hs]: I think we need a more principled solution for this.  See also https://github.com/sol/hpack/issues/219.

Comment (mizunashi-mana) [reply src/Hpack/Util.hs]: I see.

**Comment by mizunashi-mana** [comment]
OK, I close this PR.

**Pull request closed without merging**